plugins/kubernetes/README.md

```diff
@@ -186,6 +186,8 @@ To override it, set the namespace to either `default` or `kube-system` and set `
 
 ### Preventing request loss with preStop
 
+To enable the following functionality you need to set `KUBERNETES_ADD_PRESTOP=true`.
```

Comment:
Do we use `true` consistently throughout Samson? Or do we have some "boolean" ENV vars that also accept `1`, or just having the environment variable set?

Reply:
we use it in a few places ... I guess Samson::EnvCheck is the best